connectors/activemq/src/main/java/forklift/connectors/ActiveMQConnector.java

```diff
@@ -122,6 +156,13 @@ public ForkliftProducerI getTopicProducer(String name) {
     }
 
     @Override
+    public ActionSource mapSource(LogicalSource source) {
+        return source
+            .apply(RoleInputSource.class, roleSource -> new QueueSource("forklift-role-" + roleSource.getRole()))
```

Comment:
I mapped to a `QueueSource` as a light-weight default. I didn't want to use a topic, since you could have the same role running on multiple instances of forklift and you don't really want all of the roles to receive the message.

I don't know why we'd want to (but that doesn't mean it should be precluded) have multiple consumer groups for role input. I could make that work by mapping here to a `TopicSource`, and then mapping that to a `GroupedTopic` when we go to consume from role input. Is that preferable?